docs/examples/CollapsablePanel.js

```diff
@@ -0,0 +1,72 @@
+var CollapsablePanel = React.createClass({
```

Comment:
For this example the name `Panel` feels like the Bootstrap Panel. Can we rename this to `CollapsableParagraph`?